lib/bootstrap.ts

```diff
@@ -6,7 +6,7 @@ $injector.require("options", "./options");
 $injector.require("nativescript-cli", "./nativescript-cli");
 
 $injector.require("projectData", "./project-data");
-$injector.require("projectDataService", "./services/project-data-service");
+$injector.requirePublic("projectDataService", "./services/project-data-service");
```

Comment:
When you add a new method to PublicApi, you have to:
1. Update the documentation: https://github.com/NativeScript/nativescript-cli/blob/master/PublicAPI.md
2. Add the service and the exposed methods in the [nativescript-cli-lib tests](https://github.com/NativeScript/nativescript-cli/blob/master/test/nativescript-cli-lib.ts)

NOTE: More information is available [here](https://github.com/NativeScript/nativescript-cli/blob/master/PublicAPI.md#how-to-add-a-new-method-to-public-api)

Reply:
done